subjects/devops/calculator/README.md

```diff
@@ -0,0 +1,62 @@
+## Calculator
+
+### Instructions
+
+In this exercise you will make a script `calculator.sh` that will take 3 arguments, calculate the result and print it on the standard output.
+
+- The first argument and the third argument will be numbers.
+- The second argument will be the operator.
+
+Each operator should have its own function named as follow:
+- `+`: `do_add()`.
+- `-`: `do_sub()`.
+- `*`: `do_mult()`.
+- `/`: `do_divide()`.
+
+Each function will receive two arguments, the left number and the right number, and return the result of the operation.
+
+The functions assume the input is valid, so the input should be checked before to call the functions.
+
+> The functions will also be tested individually, so it is important to name each function exactly as above, the behavior of the functions have to match the exercise instructions.
+
+### Usage
+
+```console
+$ ./calculator.sh 20 "*" 3
+60
+$ ./calculator.sh 20 / 20
+1
+$ ./calculator.sh -1 - 10
+-11
+$
+```
+
+### Error handling
+
+All errors will print a specific message on **stderr** (ending with a newline) and returns a specific non-zero value:
+- Wrong number of arguments: `"Error: expect 3 arguments"`, returns `1`.
+- Division by 0: `"Error: division by 0"`, exit with `2`.
```

Comment:
This to exits are changed in the solution.
is it be like it is said in the readme or like it is in the solution?

![image](https://user-images.githubusercontent.com/53977751/211572361-59b13810-dc8d-4da0-9741-dce9da603899.png)